src/sql/workbench/parts/scripting/browser/scripting.contribution.ts

```diff
@@ -203,3 +207,103 @@ MenuRegistry.appendMenuItem(MenuId.ObjectExplorerItemContext, {
 });
 
 //#endregion
+
+//#region -- explorer widget
+
+CommandsRegistry.registerCommand(commands.ExplorerScriptSelectAction.ID, (accessor, context) => {
+	const instantiationService = accessor.get(IInstantiationService);
+	instantiationService.createInstance(commands.ExplorerScriptSelectAction, commands.ExplorerScriptSelectAction.ID, commands.ExplorerScriptSelectAction.LABEL).run(context);
+});
+
+MenuRegistry.appendMenuItem(MenuId.ExplorerWidgetContext, {
+	command: {
+		id: commands.ExplorerScriptSelectAction.ID,
+		title: commands.ExplorerScriptSelectAction.LABEL
+	},
+	when: ItemContextKey.ItemType.isEqualTo('view'),
+	order: 2
+});
+
+MenuRegistry.appendMenuItem(MenuId.ExplorerWidgetContext, {
```

Comment:
can this contribution be combined with the above using an OR condition in the when clause (e.g. for 'table' || 'view')?  They look basically the same.